src/index.js

```diff
@@ -58,26 +68,32 @@ export let ContextTree = _.curry(
     )
 
     // overwriting extend
```

Comment:
maybe `overwriting extend so it reports changes` or something

Reply:
This was your comment 😑 
https://github.com/smartprocure/contexture-client/commit/a964a30dfc33215799c6cf7dfe5f68179092971f

But I'll change it